dataset_configs/portuguese/coraa/config.yaml

```diff
@@ -0,0 +1,85 @@
+documentation: |
+  Coraa Portuguese
+  ################
+
+  The config  performs the following data processing.
+
+  1. Downloads and extracts all the data from the "https://huggingface.co/datasets/gabrielrstan/CORAA-v1.1/tree/main"
+  2. Replaces certain non-supported characters, abbreviations and  punctuation marks with equivalent supported versions.
+  3. Drops any data that contains high/low character occurence.
+  4. Drops any data that contains symbols not in the supported alphabet.
+
+  **Required arguments**.
+
+  * **workspace_dir**: specify the workspace folder where all audio files will be stored.
+  * **data_split**: should be "train", "dev" or "test".
+
+  Note that you can customize any part of this config either directly or from command-line.
+  Here are some common customizations to consider:
```

Comment:
I think this line doesn't make sense? Please remove it in this file and the other files were it appears.

Reply:
Done